src/lib/marshal/email-layout/types.ts

```diff
@@ -0,0 +1,12 @@
+import { Hyperlink, MaybeWithAnnotation } from "../shared/types";
```

Comment:
Even though we expose the command as `knock layout..`, I think given this is an internal marshal folder it makes sense to keep it as `email-layout`. It would avoid confusions compared to calling this folder only `layout` like the command, agree?